dynamicgrid_test.go

```diff
@@ -85,7 +85,12 @@ func TestDynamicGrid(t *testing.T) {
 		t.Errorf("dynamic grid should have %v cells but instead has %v", wantCells, cells)
 	}
 
-	r, err := d.Results(false, true, map[string]string{"TotalPopD": "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)"})
+	o, err := NewOutputter("", false, map[string]string{"TotalPopD": "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)"}, nil)
```

Comment:
Why does it work in the main inmap code to not return a pointer, i.e. just `o`, but to refer to the outputter struct in the test code we have to use `&o`?